### DISCUSSION TITLE
[Feature] Make argon2 password hashing parallelism option configurable

### DISCUSSION THREAD

**Discussion by ZPascal** [pretix/pretix#4641] [Ideas]
Hi community & @raphaelm ,

as you can see from this [issue](https://github.com/ZPascal/pretix-docker-compose/issues/11#issuecomment-2466148078), the Argo2 password hashing algorithm is limited to [8 CPU threads](https://github.com/django/django/blob/c56e1273a9d87ffad3a84fb597550f79b9820281/django/contrib/auth/hashers.py#L393) in the case of Django. I would like to make this option configurable, but that would require overriding the class in the Pretix source code. What do you think of this idea?

**Comment by raphaelm** [comment]
Hmm, I don't like messing with the options there, maybe rather introduce a config flag that disables argon2 alltogether and falls back to the Django default?

**Reply by raphaelm** [reply]
See https://github.com/pretix/pretix/pull/4643

**Reply by mikaoelitiana** [reply]
Thanks for the quick update
